pipeline_dp/aggregate_params.py

```diff
@@ -82,7 +79,7 @@ class AggregateParams:
     high: float = None  # deprecated
     min_value: float = None
     max_value: float = None
-    public_partitions: Any = None
+    public_partitions: Any = None  # deprecated
     noise_kind: NoiseKind = NoiseKind.LAPLACE
     custom_combiners: Iterable['CustomCombiner'] = None
 
```

Comment:
Could you please deprecate (i.e. # deprecate and check in \_\_post_init\_\_)  `public_partitions` in `SumParams`, `CountParams`, `MeanParams`, `VarienceParams` as well?